config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root to: 'users#index'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:new, :create, :show, :edit, :update]
```

Comment:
- [ ] In `config/routes.rb`, these are the only actions you need to handle. `[:new, :create, :edit, :update]`